components/BestBoards.tsx

```diff
@@ -0,0 +1,54 @@
+import getArticles, { Article } from '@/pages/api/client';
+import { useCallback, useContext, useEffect, useState } from 'react';
+import { DeviceContext } from '@/contexts/DeviceContext';
+import BestMedal from './boards/BestMedal';
+import BestContent from './boards/BestContent';
+import BestInfo from './boards/BestInfo';
+
+function BestBoards() {
+  const device = useContext(DeviceContext);
+  const [boards, setBoards] = useState<Article[]>([]);
+
+  const pageSizeByDevice = useCallback(() => {
+    if (device === 'mobile') return 1;
+    if (device === 'tablet') return 2;
+    return 3;
+  }, [device]);
+
+  useEffect(() => {
+    const handleLoad = async () => {
+      const { list } = await getArticles({
+        page: 1,
+        pageSize: pageSizeByDevice(),
+        orderBy: 'like',
+      });
+      setBoards(() => list);
```

Comment:
p4;
요건 굳이 콜백으로 작성하지 않아도 무방할 것 같아요!
```suggestion
      setBoards(list);
```